lib/icingadb/icingadb-objects.cpp

```diff
@@ -1324,7 +1324,7 @@ void IcingaDB::SendSentNotification(
 		"notification_id", GetObjectIdentifier(notification),
 		"host_id", GetObjectIdentifier(host),
 		"type", Convert::ToString(type),
-		"state", Convert::ToString(service ? Convert::ToLong(cr->GetState()) : Convert::ToLong(Host::CalculateState(cr->GetState()))),
+		"state", Convert::ToString(cr ? service ? Convert::ToLong(cr->GetState()) : Convert::ToLong(Host::CalculateState(cr->GetState())) : 99),
```

Comment:
I've already seen this magic 99 appear elsewhere in the code. Is there really no constant for this?

Reply:
Exactly. (If it already appears multiple times, I consider it another construction area.)